packages/x-components/tests/e2e/cucumber/recommendations/stubs/recommendations.stub.ts

```diff
@@ -0,0 +1,9 @@
+import { createResultStub } from '../../../../../src/__stubs__/results-stubs.factory';
+
+export default {
+  results: [
+    createResultStub('Piscina 3 Anillos'),
+    createResultStub('Among Us Figura Acción'),
+    createResultStub('Barbie Sirenas Dreamtopia')
+  ]
+};
```

Comment:
If you end up refactoring the tests using a `named export` instead of the `default` one, this should be refactored to something like:

```suggestion
export const recommendationsStub = {
  results: [
    createResultStub('Piscina 3 Anillos'),
    createResultStub('Among Us Figura Acción'),
    createResultStub('Barbie Sirenas Dreamtopia')
  ]
};
```

Reply:
The whole file is intended to be used for the recommendations stubs. Each export should be that we expected to have in the filename.